webanno-automation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/automation/service/MiraAutomationServiceImpl.java

```diff
@@ -312,18 +312,16 @@ public File getDocumentFolder(TrainingDocument trainingDocument)
     @Transactional(noRollbackFor = NoResultException.class)
     public List<TrainingDocument> listTrainingDocuments(Project aProject)
     {
-        // both TAB_SEP and WebAnno training documents
-        List<TrainingDocument> trainingDocuments = entityManager
-                .createQuery("FROM TrainingDocument where project =:project",
-                        TrainingDocument.class)
-                .setParameter("project", aProject).getResultList();
         /*
          * List<TrainingDocument> webAnnoTraiingDocuments = new ArrayList<TrainingDocument>(); for
          * (TrainingDocument trainingDocument : trainingDocuments) { if
          * (trainingDocument.getFormat().equals(WebAnnoConst.TAB_SEP)) {
          * webAnnoTraiingDocuments.add(trainingDocument); } }
          */
-        return trainingDocuments;
+
```

Comment:
I actually like keeping these variables because they allow inspecting the value in the debugger before it is being returned. In particular if the returned collection is then used in an extend for-loop, it is difficult to see the contents of the collection without this variable.